CWE/Modules/AutoRolesModule.cs

```diff
@@ -59,7 +59,7 @@ public async Task AutoRoles()
                     .WithStyle(EmbedStyle.Error)
                     .Build();
 
-                await this.Context.Channel.SendMessageAsync(embed: noRanks);
+                await Context.Channel.SendMessageAsync(embed: noRanks);
```

Comment:
Are we standardising on `SendMessageAsync` for non-logging messages, or should they be returned using `ReplyAsync`?

Reply:
Yes, we can switch back to ReplyAsync, changed in https://github.com/Coding-with-Efehan/CWE/pull/16/commits/24e6058a1e7cd0316e486c437dc6499eb3d45f73.